apps/style/code-studio/curriculum_catalog_card.scss

```diff
@@ -0,0 +1,65 @@
+@import "phase1-design-system";
+
+$border-radius: 4px;
+
+.curriculumCatalogCardContainer {
+  display: flex;
+  flex-flow: column;
+  background-color: $neutral_light;
+  border-radius: $border-radius;
+  height: 384px;
+  width: 309px;
+
+  img {
+    height: 140px;
+    object-fit: cover;
+    border-radius: $border-radius $border-radius 0 0;
+  }
+
+  .curriculumInfoContainer {
+    display: flex;
+    flex-flow: column;
+    height: 100%;
+    border: 1px solid $neutral_dark20;
+    border-top: 0;
+    border-radius: 0 0 $border-radius $border-radius;
+    padding: 1.25em 1em;
+
+    .overline {
+      color: $brand_primary_default;
+      margin-bottom: 0.5em;
+    }
+
+    h4 {
+      margin-bottom: 0.75em;
+      color: $neutral_dark;
+    }
+
+    .iconWithDescription {
+      display: flex;
+      color: $neutral_dark80;
+      margin: 0 0 0.5em 0;
+      align-items: center;
+
+      .iconDescription {
+        font-size: 0.875em;
+        margin: 0 0 0 0.5em;
+        color: $neutral_dark;
+      }
+    }
+
+    .buttonsContainer {
+      display: flex;
+      flex-wrap: wrap;
+      justify-content: space-between;
+      margin-top: auto;
+
+      button {
+        height: 40px;
+        margin: 0;
+        width: 132px; // calculated: (totalContainerWidth - 2*padding - minimumSpaceBetweenButtons) / 2
```

Comment:
This will need to change to accommodate for longer strings –– see follow-up tasks for more details

Reply:
not sure it's the right place in code to comment BUT
btw, another option I thought of after the call - you can hardcode to always render **3** (for example) topics and in case there're more than 3 topics -> render `+n `instead of the third topic

Reply:
Yep –– that might be a good fallback for now. Thanks!